cogs/utils/auto_incorrect.py

```diff
@@ -0,0 +1,73 @@
+import sys, random
+
+
+# Credit to lazho for this function
+def auto_incorrect(input_str):
+    chance = 99
+    input_str = input_str.split()
+
+    specialCases = {
+        "your": "you're",
+        "you're": "your",
+        "its": "it's",
+        "it's": "its",
+        "their": "they're",
+        "they're": "there",
+        "there": "their",
+        "lose": "loose",
+        "loose": "lose",
+        "effect": "affect",
+        "affect": "effect",
+        "definitely": "definately",
+        "weather": "whether",
+        "whether": "weather",
+        "then": "than",
+        "an": "a",
+        "the": "teh",
+        "like": "liek"
+    }
```

Comment:
A few other ones you can add maybe- "fucking": "ducking", "sick": "dick", "gum": "cum"